src/xc_integrator/local_work_driver/device/common/device_blas.hpp

```diff
@@ -63,5 +63,10 @@ void syr2k( device_blas_handle handle,
             int M, int K, T ALPHA, 
             const T* A, int LDA, const T* B, int LDB,
             T BETA, T* C, int LDC );
+
+template <typename T>
```

Comment:
This is already present in `device_backend`: https://github.com/wavefunction91/GauXC/blob/b0f2d7e7b332a5fbaa5a15125a42d23c7c724456/src/runtime_environment/device/device_backend.hpp#L54-L58. Note the changes that need to be made in the function calls based on the type-dependent offsets